src/domain/models/VerifiableCredential.ts

```diff
@@ -138,6 +193,21 @@ export type PresentationSubmission = {
   verifiablePresentation: string[],
 }
 
+export type AnoncredsPresentationSubmission = Anoncreds.PresentationType;
+
+
+
+export type PresentationSubmissionData = {
+  [CredentialType.AnonCreds]: AnoncredsPresentationSubmission;
+  [CredentialType.JWT]: JWTPresentationSubmission;
+  [CredentialType.Unknown]: any;
+  [CredentialType.W3C]: any;
+}
```

Comment:
Because of the CredentialType inference, I require to at least have defined each one of the properties that CredentialType has and set a value for those, but they are not used at all. 

I'm resolving this one :)